engine/src/flutter/lib/web_ui/lib/src/engine/frame_reference.dart

```diff
@@ -2,10 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-/// A monotonically increasing frame number being rendered.
-///
-/// Used for debugging only.
-int debugFrameNumber = 1;
+// TODO(yjbanov): this file should be deleted as part of https://github.com/flutter/flutter/issues/145954
```

Comment:
Nice catch! I think it's better if we moved the file so that it gets deleted with the html code: https://github.com/flutter/flutter/pull/162608